installer/README.md

```diff
@@ -1,29 +1,71 @@
-# Introduction
-This is a standalone/non-containerized installer for SODA Infrastructure Manager (delfin) project.
-It contains a script and options to check the environment feasible for installing delfin. Installs required dependent software/binaries.
+# Delfin Installation Guide
+
+The SODA Delfin supports two types of installation
+* Installation using Ansible for user experiance with Dashboard
+* Installation using scripts
 
-# Supported OS
-Ubuntu 16.04, Ubuntu 18.04
+## Ansible installer
 
-# Prerequisite
+* Supported OS: **Ubuntu 20.04, Ubuntu 18.04**
+* Prerequisite: **Python 3.6 or above** should be installed
 
-- #### Ensure the logged-in user has root privileges.
+### Install steps
+
+```bash
+sudo apt-get update && sudo apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk
+git clone https://github.com/sodafoundation/delfin.git
+# git checkout <delfin-release-version>
+cd delfin/installer
+chmod +x install_dependencies.sh && sudo ./install_dependencies.sh
+cd ansible
+export PATH=$PATH:/home/$USER/.local/bin
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts -v
+```
+### Uninstall
+```bash
+sudo -E env "PATH=$PATH" ansible-playbook clean.yml -i local.hosts -v
+```
 
-- #### Setup Python3
-    Python3 and Pip3 should be installed on the system.
 
-    Note: If you don't have python3 in your system, you may follow below steps to setup python3 environment.
+### Logs
+Delfin processes execution logs can be found in /tmp/ folder
+* /tmp/api.log
+* /tmp/alert.log
+* /tmp/task.log
+* /tmp/exporter.log
+* /tmp/create_db.log
 
-    ```sh
-    apt-get install python3
-    apt-get install python3-pip
-    ```
+### How to use Delfin
+Delfin can be used either through dashboard or REST APIs.
+
+Please refer [user guides](https://docs.sodafoundation.io/guides/user-guides/delfin/dashboard/)
+
+
+
+## Bash installer
```

Comment:
Can we use the installer names consistently in the beginning also? Or please say Installation option 1 : Ansible...and Installation option 2: Bash Script installer in the title. Also use consistently across the doc

Reply:
Done